editoast/src/views/v2/train_schedule.rs

```diff
@@ -682,41 +683,104 @@ pub async fn project_path(
 #[derive(Debug, Serialize, ToSchema)]
 enum SimulationSummaryResultResponse {
     // Minimal information on a simulation's result
-    #[allow(dead_code)]
     Success {
         // Length of a path
         length: u64,
         // travel time
-        time: f64,
+        time: u64,
```

Comment:
Can you specify the unit in the comment above (same for `length` and `energy_consumption` too)🙏